collect_app/src/main/java/org/odk/collect/android/tasks/sms/SmsUtils.java

```diff
@@ -0,0 +1,160 @@
+package org.odk.collect.android.tasks.sms;
+
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+
+import org.odk.collect.android.R;
+import org.odk.collect.android.events.SmsEvent;
+import org.odk.collect.android.tasks.sms.models.MessageStatus;
+import org.odk.collect.android.tasks.sms.models.SmsProgress;
+import org.odk.collect.android.tasks.sms.models.SmsSubmission;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+public class SmsUtils {
```

Comment:
Can these methods find homes in the new abstractions built? Adding a utils class should really be a last resort when either a new abstraction isn't clear or it would take too much restructuring at the moment.